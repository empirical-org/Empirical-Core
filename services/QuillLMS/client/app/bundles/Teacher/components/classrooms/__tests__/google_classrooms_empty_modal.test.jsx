import { shallow } from 'enzyme';
import React from 'react';

import GoogleClassroomsEmptyModal from '../google_classrooms_empty_modal';

describe('GoogleClassroomsEmptyModal component', () => {

  const wrapper = shallow(
    <GoogleClassroomsEmptyModal
      close={() => {}}
    />
  );

  it('should render GoogleClassroomsEmptyModal', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
