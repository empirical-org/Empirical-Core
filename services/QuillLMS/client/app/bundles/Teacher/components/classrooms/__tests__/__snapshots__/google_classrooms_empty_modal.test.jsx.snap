// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleClassroomsEmptyModal component should render GoogleClassroomsEmptyModal 1`] = `
<div
  className="modal-container google-classrooms-empty-modal-container"
>
  <div
    className="modal-background"
  />
  <div
    className="google-classrooms-empty-modal quill-modal modal-body"
  >
    <div>
      <h3
        className="title"
      >
        Create classes in Google Classroom
      </h3>
    </div>
    <div
      className="no-active-classes"
    >
      <img
        alt="empty class"
        src="undefined/images/illustrations/empty-class.svg"
      />
      <p>
        No classes are available to import yet. Go to Google Classroom and create classes to import them into Quill.
      </p>
    </div>
    <div
      className="form-buttons"
    >
      <button
        className="quill-button contained primary medium"
        onClick={[Function]}
        type="button"
      >
        Close
      </button>
    </div>
  </div>
</div>
`;
